liquidctl/driver/smart_device.py

```diff
@@ -653,7 +653,7 @@ def _write_colors(self, cid, mode, colors, sval, direction='forward',):
                             0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x06,
                             0x05, 0x85, 0x05, 0x85, 0x05, 0x85, 0x00, 0x00,
                             0x00, 0x00, 0x00, 0x00])
-                    self._write(msg + color_lists[i % 4])
+                    self._write(msg + color_lists[i % 3])
```

Comment:
For reasons unknown to me (don't have this particular device), it seems that the modulo 4 indexing was intentional, and that some writes should send an empty color list.

```suggestion
                    self._write(msg + color_lists[i % 4])
```